app/views/papers/_form.html.erb

```diff
@@ -11,113 +11,136 @@
-      reporting
-    </label></br>
-    <label>
-      <%= f.check_box :bias_list, {multiple: true}, 'small sample size', nil %>
-      small sample size
-    </label>
-  </div>
+      <div class="form-group">
+        <%= f.label :methodlogy_list, class: "col-md-2 control-label"  %>
+        <div class="col-md-10">
+          <label class= "checkbox-inline">
+            <%= f.check_box :methodology_list, {multiple: true}, 'meta-analysis', nil %>
+            meta-analysis
+          </label>
+          <label class= "checkbox-inline">
+            <%= f.check_box :methodology_list, {multiple: true}, 'systematic review', nil %>
+            systematic review
+          </label>
+          <label class= "checkbox-inline">
+            <%= f.check_box :methodology_list, {multiple: true}, 'randomized control trial', nil %>
+            randomized control trial
+          </label>
+          <label class= "checkbox-inline">
+            <%= f.check_box :methodology_list, {multiple: true}, 'quasi-experiment', nil %>
+            quasi-experiment
+          </label>
+          <label class= "checkbox-inline">
+            <%= f.check_box :methodology_list, {multiple: true}, 'cohort', nil %>
+            cohort
+          </label>
+          <label class= "checkbox-inline">
+            <%= f.check_box :methodology_list, {multiple: true}, 'case-control', nil %>
+            case-control
+          </label>
+          <label class= "checkbox-inline">
+            <%= f.check_box :methodology_list, {multiple: true}, 'cross-sectional survey', nil %>
+            cross-sectional survey
+          </label>
+          <label class= "checkbox-inline">
+            <%= f.check_box :methodology_list, {multiple: true}, 'case report', nil %>
+            case report
+          </label>
+        </div>
+      </div>
+      <div class="form-group">
+        <%= f.label :bias_list, class: "col-md-2 control-label"  %>
+          <div class="col-md-10">
+          <label class= "checkbox-inline">
+            <%= f.check_box :bias_list, {multiple: true}, 'selection', nil %>
+            selection
+          </label>
+          <label class= "checkbox-inline">
+            <%= f.check_box :bias_list, {multiple: true}, 'performance', nil %>
+            performance
+          </label>
+          <label class= "checkbox-inline">
+            <%= f.check_box :bias_list, {multiple: true}, 'detection', nil %>
+            detection
+          </label>
+          <label class= "checkbox-inline">
+            <%= f.check_box :bias_list, {multiple: true}, 'attrition', nil %>
+            attrition
+          </label>
+          <label class= "checkbox-inline">
+            <%= f.check_box :bias_list, {multiple: true}, 'reporting', nil %>
+            reporting
+          </label>
+          <label class= "checkbox-inline">
+            <%= f.check_box :bias_list, {multiple: true}, 'small sample size', nil %>
+            small sample size
+          </label>
```

Comment:
Can you generate these from the backend? That is, do we have the list of tags somewhere?